src/sass/_functions.scss

```diff
@@ -229,3 +229,40 @@ Will generate for example following classes:
         @include createSelector(#{$newSelector}, $properties);
     }
 }
+
+// @include breakpoint('xs') { width: 90%; }
+@mixin breakpoint($bpName, $breakpoints: $breakpoints) {
+    $bpValue: map-get($breakpoints, $bpName);
+    @media (min-width: $bpValue) { @content; }
+}
+
+@mixin createClasses($map) {
+    @each $selector, $definition in $map {
+        .#{$selector} {
+            @each $property, $value in $definition {
+                #{$property}: $value;
+            }
+        }
+    }
+}
+
+@mixin responsiveClasses($selector, $_breakpoints: $breakpoints) {
+    @each $name, $value in $breakpoints {
+        $responsiveSelector: 'r-' + $name + '-' + $selector;
```

Comment:
How about adding the selector naming scheme as a parameter? So that if user don't like Once naming convention, they can change it to whatever they like.

```
$responsiveClassScheme: '{r-{breakpoint}-{class}';
@mixin responsiveClasses($selector, $_responsiveClassScheme: $responsiveClassScheme, $_breakpoints: $breakpoints) 
```